.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf 
```

Comment:
no carriage returns?! ;)

Reply:
Normally I wouldn't even care about this but it's important for Windows users to use LF when working with macOS/linux

Reply:
It's important for everyone to kill the carriage return because we don't type on fucking typewriters.

Reply:
Speak for yourself. I love my typewriter 😤
![image](https://user-images.githubusercontent.com/9084735/110271515-3969f780-7f96-11eb-9109-fa81bb6192ea.png)